src/modules/grubcfg/main.py

```diff
@@ -51,20 +55,26 @@ def modify_grub_default(partitions, root_mount_point, distributor):
             if partition["fs"] == "linuxswap":
                 swap_uuid = partition["uuid"]
 
-            if partition["fs"] == "linuxswap" and "luksMapperName" in partition:
+            if partition[
+                    "fs"] == "linuxswap" and "luksMapperName" in partition:
                 swap_outer_uuid = partition["luksUuid"]
```

Comment:
ok, you are right again - and afaik using a prefix operator is preferred in pycodestyle, it was postfix in pep8. But it might be that i'm not remembering this right.